### ISSUE TITLE
Doesn't support multiple files?

### ISSUE THREAD

**Issue by QadiymStewart** [DataJuggler/BlazorFileUpload#10]
Does this support uploading multiple files at once?  "multiple" html input tag like the original from steves files input

**Comment by DataJuggler** [comment]
No, to answer your question. I guess it is a matter of personal preference, but I tend to like to start  as simple as possible at first or the risk of 'what can go wrong' increases as you add more moving parts.

If you are wanting this feature and / or volunteering if you feel you are up for it, I will update the Nuget package.  I saw Steve's example with the multiple files once, just never dove in and tried it.

Besides the multiple parameter, do you happen to know if more than one file stream is returned, is that how he does it? I just worked on the code a few days ago and added a SaveToDisk parameter, and if false the MemoryStream is returned in UploadFileInfo, or are more than one UploadFileInfo instances returned?

I tend to ask questions when I could I could just run the program and find out in almost the same time.

I will try and work on this over the weekend.

Thanks, for the feedback. 

Corby

**Comment by araujofrancisco** [comment]
I did an initial implementation for multiple files, so far it seems to be working. You are welcome to test it and report any issues.

**Comment by DataJuggler** [comment]
I will test it when I get a chance.

I wrote BlazorFileUpload for my site https://pixeldatabase.net , and I only upload 1 file at a time, but I know others will appreciate it.

I have some plans for other sites this would come in handy.

It has been on my to do list to build a progress bar, so I will work on that soon.

Thanks,

Corby

**Comment by hericdenis** [comment]
> I did an initial implementation for multiple files, so far it seems to be working. You are welcome to test it and report any issues.

Thanks for the fork dude! I tried to implement it myself in a similar way to yours but kept getting exceptions for some odd reason.
You really saved my skin hahaha

**Comment by araujofrancisco** [comment]
> > I did an initial implementation for multiple files, so far it seems to be working. You are welcome to test it and report any issues.
> 
> Thanks for the fork dude! I tried to implement it myself in a similar way to yours but kept getting exceptions for some odd reason.
> You really saved my skin hahaha

You are welcome! Let me know if you find out any issues.
